v2/plugins/redhat/project-initializer/0.0.9/meta.yaml

```diff
@@ -0,0 +1,12 @@
+publisher: redhat
+type: VS Code extension
+name: project-initializer
+title: Project Initializer by Red Hat
+version: 0.0.9
+displayName: Project Initializer by Red Hat
+description: A lightweight extension based on Red Hat launcher to generate quickstart projects using Eclipse Che
+url: https://download.jboss.org/jbosstools/adapters/snapshots/vscode-project-initializer/project_initializer-0.0.9-92.vsix

```

Comment:
@jmaury is it possible to have a more reliable URL?

this one is in Snapshot folder, I fear it can vanish at any time.

The marketplace URL needs to be avoided due to VS Code Marketplace rate limit error https://github.com/eclipse/che/issues/12840

The stable folder doesn't contain the specific version for project initializer (it contains specific versions for other extensions)
![image](https://user-images.githubusercontent.com/1105127/57213770-18c26680-6fe8-11e9-9c6c-234b47392541.png)

Even this "stable" folder seems not completely reliable because it is under "adapters" folder.

Reply:
Updated there: you can access latest version from http://download.jboss.org/jbosstools/adapters/stable/vscode-project-initializer/project_initializer-0.0.9-92.vsix

Reply:
I updated with the new URL, thanks!